components/eam/bld/namelist_files/namelist_defaults_eam.xml

```diff
@@ -426,7 +426,12 @@
 <flbc_file chem="waccm_mozart_sulfur">atm/waccm/lb/LBC_1765-2005_1.9x2.5_CMIP5_za_c111110.nc</flbc_file>
 
 <!-- Time-variant CO2 fossil fuel emissions -->
-<co2flux_fuel_file  sim_year="1850-2000">atm/cam/ggas/co2flux_fossil_1751-2006-monthly_0.9x1.25_c20100204.nc</co2flux_fuel_file>
+<!-- <co2flux_fuel_file  sim_year="1850-2000">atm/cam/ggas/co2flux_fossil_1751-2006-monthly_0.9x1.25_c20100204.nc</co2flux_fuel_file> -->
```

Comment:
The changes here and in the following lines update the CO2 emission files - do they belong in this mass conservation PR?

Reply:
Same as before; this is not needed.